### ISSUE TITLE
feat(table): localize aria-sort attribute

### ISSUE THREAD

**Issue by gonzalad** [angular/components#30398]
### Feature Description

The aria-sort value is hardcoded in MatSortHeader  to none, ascending or descending.

Could the material library provide a way to localize its value (i.e. german, french, etc...) ?

https://github.com/angular/components/blob/a08eeebf6b334e00a491496829e3838d0616258a/src/material/sort/sort-header.ts#L242-L248

**Comment by crisbeto** [comment]
[Via MDN](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-sort):

> If a grid or table provides sort functionality, the aria-sort attribute should be set to either ascending or descending (or other) on the header cell element for the sorted column or row.


My understanding is that it's up to the screen reader to localize it.
